frontend/packages/dev-console/integration-tests/features/guided-tour/quick-starts/quick-start-access-code-snippet.feature

```diff
@@ -27,11 +27,10 @@ Feature: Access a code snippet from a quick start
 
 
```

Comment:
```suggestion
And getting started with spring CR Quick Start is available
```

Reply:
Update line 8

Reply:
No, we have sample-application CR in the background statement which gives `Getting started with a sample application` quick start